testapps/BeautifulSoup4/main.py

```diff
@@ -1,128 +1,67 @@
 from kivy.app import App
-from kivy.uix.button import Button 
-from kivy.uix.textinput import TextInput 
-from kivy.uix.label import Label 
+from kivy.uix.button import Button
+from kivy.uix.label import Label
 from kivy.uix.boxlayout import BoxLayout
-from kivy.uix.gridlayout import GridLayout
 from kivy.lang import Builder
 from kivy.uix.popup import Popup
-from kivy.uix.image import AsyncImage 
 
 import requests
-from bs4 import BeautifulSoup 
+from bs4 import BeautifulSoup
 
-Builder.load_string(
-'''
+Builder.load_string('''
 <PopupScreen>:
-	orientation: 'vertical'
-	Label:
-		text: "Enter Your movie name buddy"
-	TextInput:
-		id: name
-		font_size: 40
-		multiline: False
-	Button:
-		text: "show details"
-		on_press: root.detail(name.text)
-
+    orientation: 'vertical'
+    Button:
+        text: "Show Story"
+        on_press: root.detail()
+        size_hint_y: None
+        height: '40dp'
+    Label:
+        id: title
+        font_size: 30
+        size_hint_y: None
+        height: '50dp'
+    ScrollView:
+        Label:
+            id: story  
+            font_size: 20
+            text_size: self.width, None
+            size_hint_y: None
+            height: self.texture_size[1]
 ''')
-global writer
-writer = "Writers:"
-global actor
-actor = "Actors:"
-global director
-director = "Directors:"
-class PopupScreen(BoxLayout):
-	def error(self):
-		popup = Popup(title="ERROR",content=Label(text="Bro, Check your movie name"),size_hint=(0.8,0.8))
-		popup.open()
-
-	def Pop(self,movie,imdb,actor,director,writer,url):
-		wid1 = GridLayout(cols=2,rows=1)
-		lb = AsyncImage(source=url)
-		wid1.add_widget(lb)
-		wid2 = GridLayout(rows=4)
-		lb = Label(text=imdb)
-		wid2.add_widget(lb)
-		
-		lb = Label(text=actor)
-		wid2.add_widget(lb)
-		lb = Label(text=director)
-		wid2.add_widget(lb)
-		lb = Label(text=writer)
-		wid2.add_widget(lb)
-
-		wid1.add_widget(wid2)		
-		popup = Popup(title=movie,content=wid1,size_hint=(0.8,0.8))
-		popup.open()
-
-	def detail(self, title):
-
-		global writer
-		writer = "Writers:"
-		global actor
-		actor = "Actors:"
-		global director
-		director = "Directors:"
 
-		s="+".join(title.split())
 
+class PopupScreen(BoxLayout):
+    def error(self):
+        popup = Popup(title="ERROR",
+            content=Label(text="Check Your Network Connection"),
```

Comment:
If you must catch exceptions in this example, please show the exception message instead of assuming what the error is.